recipes/mob_suite/setup.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index d6c8f9e..b905b11 100644
+--- a/setup.py
++++ b/setup.py
+@@ -58,7 +58,7 @@ setup(
+ 
+     entry_points={
+         'console_scripts': [
+-            'Sequenoscope=Sequenoscope:main'
++            'Sequenoscope=Sequenoscope:main',
```

Comment:
Nice catch, do you intend to submit this to the upstream project?

Reply:
Yes, sorry for the delay. I've opened a PR to the upstream project: [phac-nml/mob-suite/#147](https://github.com/phac-nml/mob-suite/pull/147).